nibabel/gifti/gifti.py

```diff
@@ -46,17 +47,14 @@ def get_metadata(self):
         return self.data_as_dict
 
     def to_xml(self):
-        if len(self.data) == 0:
-            return "<MetaData/>\n"
-        res = "<MetaData>\n"
+        metadata = xml.Element('MetaData')
         for ele in self.data:
-            nvpair = """<MD>
-\t<Name><![CDATA[%s]]></Name>
-\t<Value><![CDATA[%s]]></Value>
-</MD>\n""" % (ele.name, ele.value)
-            res = res + nvpair
-        res = res + "</MetaData>\n"
-        return res
+            md = xml.SubElement(metadata, 'MD')
+            name = xml.SubElement(md, 'Name')
+            value = xml.SubElement(md, 'Value')
+            name.text = ele.name
+            value.text = ele.value
```

Comment:
I believe the `CDATA` bit is a way to avoid encoding strings as XML-safe. That trick isn't necessary; the library will XML encode.

In addition, if you do as you suggest, I believe the `<` and `>` characters will be escaped, and you'll get something you don't want.